crates/evm/src/genesis.rs

```diff
@@ -168,7 +168,7 @@ impl<C: sov_modules_api::Context> Evm<C> {
             );
 
             if acc.code.len() > 0 {
-                evm_db.insert_code(acc.code_hash, acc.code.clone());
+                evm_db.insert_code(acc.code_hash, Bytecode::new_raw(acc.code.clone()));
```

Comment:
Bytecode has function `hash_slow`

let's  use that in above lines instead of keccak256(acc.code)